ml-agents/mlagents/trainers/cli_utils.py

```diff
@@ -171,14 +186,14 @@ def _create_parser() -> argparse.ArgumentParser:
     argparser.add_argument(
         "--torch",
         default=False,
-        action=DetectDefaultStoreTrue,
-        help="Use the PyTorch framework. Note that this option is not required anymore as PyTorch is the"
+        action=RaiseDeprecationWarning,
+        help="(Deprecated) Use the PyTorch framework. Note that this option is not required anymore as PyTorch is the"
         "default framework, and will be removed in the next release.",
     )
     argparser.add_argument(
         "--tensorflow",
         default=False,
-        action=DetectDefaultStoreTrue,
+        action=RaiseDeprecationWarning,
```

Comment:
Should we remove this one altogether? (Deprecated) usually means that it still works but will be removed in the future, whereas this flag won't do anything

Reply:
I see 3 options:
 - Remove --tensorflow and --torch (I think users will complain that a command line that used to work does not anymore)
 - Raise a specific error when --tensorflow is used to signal to the user that the argument has been removed
 - Only raise a warning when used (the warning can say "feature removed" instead of "feature deprecated"
I have a preference for option 2 but not a strong one. What do you think?